docs/manuals/source/DocumentationStyleGuide/BareosSpecificFormatting/Links.rst

```diff
@@ -0,0 +1,71 @@
+Links
```

Comment:
Maybe a Link to this section should be placed also in RestOverview.html#hyperlinks (Bareos links)

Reply:
Done.